draft-packages/select/KaizenDraft/Select/styles.react.scss

```diff
@@ -90,34 +90,80 @@
     cursor: default;
   }
 
-  &.secondary.reversed {
-    .control {
-      background: add-alpha($kz-color-white, 10%);
-      color: $kz-color-white;
-      border: $kz-border-borderless-border-width
-        $kz-border-borderless-border-style $kz-border-borderless-border-color;
-      border-radius: $kz-border-borderless-border-radius;
-
-      &:hover {
-        background: add-alpha($kz-color-white, 20%);
+  &.secondary {
+    &.default {
+      .control {
+        background: transparent;
+        border: $kz-border-borderless-border-width
+          $kz-border-borderless-border-style $kz-border-borderless-border-color;
+        border-radius: $kz-border-borderless-border-radius;
+
+        .singleValue {
+          color: $kz-color-cluny-500;
+        }
+
+        &:hover {
+          background: $kz-color-cluny-100;
+        }
+
+        &.disabled {
+          opacity: 0.3;
+        }
+
+        &.focusedControl {
+          background: $kz-color-cluny-100;
+
+          &::after {
+            $focus-ring-offset: ($kz-border-focus-ring-border-width * 2) + 1px;
+            content: "";
+            position: absolute;
+            background: transparent;
+            border-radius: $kz-border-focus-ring-border-radius;
+            border-width: $kz-border-focus-ring-border-width;
+            border-style: $kz-border-focus-ring-border-style;
+            border-color: $kz-color-cluny-500;
+            top: -$focus-ring-offset;
+            left: -$focus-ring-offset;
+            right: -$focus-ring-offset;
+            bottom: -$focus-ring-offset;
+          }
+        }
       }
 
-      &.disabled {
-        opacity: 0.3;
+      .dropdownIndicator {
+        color: $kz-color-cluny-500;
       }
     }
 
-    .dropdownIndicator {
-      color: $kz-color-white;
-    }
+    &.reversed {
+      .control {
+        background: add-alpha($kz-color-white, 10%);
+        color: $kz-color-white;
+        border: $kz-border-borderless-border-width
+          $kz-border-borderless-border-style $kz-border-borderless-border-color;
+        border-radius: $kz-border-borderless-border-radius;
+
+        &:hover {
+          background: add-alpha($kz-color-white, 20%);
```

Comment:
This sparked my question around how opacity should be achieved. The consensus is to use `add-alpha($kz-color-white, 20%)` (it's a the scss function). I realise this is just inherited code, so no need to update it here. We could do a big sweep across the codebase easily enough.

Reply:
> The consensus is to use add-alpha($kz-color-white, 20%) (it's a the scss function)

The policy that we've been communicating/using for the last few months has been `rgba($kz-color-white, 20%).` We've been telling people to stop using `add-alpha` since it's a custom function that I built and we want to only use built-in SCSS functions.

Reply:
(I agree it sucks that it's not real CSS – having unexpected bugs because of the build not breaking when custom Sass functions aren't present seems worse though!)